### PR TITLE
Append username to default ocs installations

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: tests/test_ocs_basic_install.py [hunk 1]
```diff
@@ -34,7 +35,8 @@ def run(**kwargs):
     log.info("Generating install-config")
     # TODO: determine better place to create cluster directories - (log dir?)
     cluster_dir_parent = "/tmp"
-    base_name = test_data.get('cluster-name', 'ocs-ci-cluster')
+    default_cluster_name = f"ocs-ci-cluster-{getuser()}"
```
Comment (clacroix12) [inline tests/test_ocs_basic_install.py]: I think we can probably remove 'cluster' from the default string at this point. `ocs-ci-<username>` seems sufficient to me, and the name does get a bit unwieldy once we add our unique number as well as the unique string the installer adds.

**Comment by petr-balogh** [comment]
This default cluster name is changed and moved to defaults in this PR https://github.com/red-hat-storage/ocs-ci/pull/27, so I hoped it will be approved from you @clacroix12 @vasukulkarni and we can merge tomorrow, but if you will sooner merge this one I will have to rebase, solve the conflict and wait till another day to get review, so if possible wait with this and rebase once we merge mentioned pr. Thanks

**Comment by vasukulkarni** [comment]
@petr-balogh no worries, I approved that one, this pr will be rebased and merged after #27

File: tests/test_ocs_basic_install.py [hunk 2]
```diff
@@ -34,7 +35,8 @@ def run(**kwargs):
     log.info("Generating install-config")
     # TODO: determine better place to create cluster directories - (log dir?)
     cluster_dir_parent = "/tmp"
-    base_name = test_data.get('cluster-name', 'ocs-ci-cluster')
+    default_cluster_name = f"ocs-ci-{getuser()}"
```
Comment (petr-balogh) [inline tests/test_ocs_basic_install.py]: @vasukulkarni can you please rebase and probably move this to defaults as it was moved there in #27 , thanks

**Pull request merged**